src/app/errorStylingandMessages.tsx

```diff
@@ -0,0 +1,34 @@
+import Paper from "@mui/material/Paper";
```

Comment:
might be good to have "use client" here 
(though it's technically implicitly set to client side already)